docs/API/Users_Guide/Submission.md

```diff
@@ -2319,6 +2319,71 @@ curl --header "X-Auth-Token: $token" 'https://api.gdc.cancer.gov/v0/submission/T
 
 Submitters can use the GraphQL query language for advanced search and retrieval of data from the GDC Submission Portal. See [GraphQL](#querying-submitted-data-using-graphql) for more information.
 
+## Patching Entitites
+
+The GDC Submission API supports the HTTP PATCH method for updating existing entities with additional fields.
+
+**PATCH** can be used to add extra fields to an existing entity, without requiring the submission of required fields. 
+
+The PATCH method cannot be used to create new entities, and the provided submitter_id must match an existing submitter_id. 
+
+```Request1
+{
+  "project_id": "GDC-INTERNAL",
+  "type": "demographic",
+  "submitter_id": "demographic7892",
+  "cause_of_death": "Infection", 
+  "cause_of_death_source": "Death Certificate", 
+  "country_of_birth": "Antigua and Barbuda", 
+  "country_of_residence_at_enrollment": "Antigua and Barbuda"
+   "projects": {
```

Comment:
My guess is that you did test this exact JSON right?  The demographic field should not have a link to project (it should be to cases), but if this worked, then maybe the entire "projects" section can be removed.

Reply:
Good catch - this was leftover from a section I copied and pasted previously. Removed projects section